great_expectations/data_context/types/base.py

```diff
@@ -2151,6 +2151,10 @@ def get_schema_class(cls):  # type: ignore[no-untyped-def]
     def config_version(self):
         return self._config_version
 
+    @config_version.setter
+    def config_version(self, config_version: float) -> None:
+        self._config_version = config_version
```

Comment:
Because of the property above, we need an explicit setter here to fully enable read/write access for all attributes on this config object.